data/BMayowa.json

```diff
@@ -0,0 +1,65 @@
+{
+	"name": "Mayowa Obisesan",
+	"displayStatsPublic": true,
+	"type": "personal",
+	"bio": "Software Engineer | Built [createsummary.com](https://www.createsummary.com) | [Eye CSS](https://github.com/BMayowa/EYE_CSS)",
+	"tags": [
+		"Python",
+		"JavaScript",
+		"Django",
+		"FastAPI",
+		"Vue JS",
+		"NodeJS",
+		"CSS",
+		"AWS | DigitalOcean"
+	],
+	"socials": [{
+			"icon": "FaTwitter",
+			"url": "https://www.twitter.com/blessed_mayowa"
+		},
+		{
+			"icon": "FaGithub",
+			"url": "https://www.github.com/BMayowa"
+		}
+	],
+	"testimonials": [],
+	"links": [{
+			"name": "Portfolio",
+			"url": "https://bmayowa.github.io",
+			"icon": "FaLink"
+		},
+		{
+			"name": "Twitter",
+			"url": "https://twitter.com/blessed_mayowa",
+			"icon": "FaTwitter",
+			"color": "#00ACEE"
+		},
+		{
+			"name": "GitHub",
+			"url": "https://github.com/BMayowa",
+			"icon": "FaGithub"
+		},
+		{
+			"name": "LinkedIn",
+			"url": "https://linkedin.com/in/mayowa-obisesan",
+			"icon": "FaLinkedin"
+		}
+	],
+	"milestones": [{
+			"title": "Released Summary",
+			"date": "November 30, 2022",
+			"icon": "",
```

Comment:
I think you should add an icon. If you're not going to use an icon, remove the field, then.

Reply:
You make a valid point, but this can come in a future PR